### DISCUSSION TITLE
Print pdf from apitemplate.io

### DISCUSSION THREAD

**Discussion by magiaslab** [Budibase/budibase#5902] [Help]
I am trying to get a pdf from the view of a row of a table to print using apitemplate.io which I already use. 
I created a REST source and pass the data correctly via POST using a test json. 
However, I cannot connect the data of the form _id of the view or even just pass a selected row of the table. What is the correct syntax to select table and pass fields as json. 
![CleanShot 2022-05-16 at 09 45 17](https://user-images.githubusercontent.com/9402517/168543739-2b3ac818-733a-42dc-8a2f-36514713cdb2.png)
![CleanShot 2022-05-16 at 09 44 50](https://user-images.githubusercontent.com/9402517/168543742-74c68ed3-3b71-456b-8924-5dc8e51cc1cd.png)
I attach the screenshots.

**Comment by melohagan** [comment]
Hi @magiaslab 

You can pass through bindings to your REST queries: https://docs.budibase.com/docs/rest-bindings

You need to make sure that the query has bindings available e.g.
![Screenshot 2022-05-16 at 14 24 27](https://user-images.githubusercontent.com/101575380/168602719-745d0452-25e8-4439-9ac6-5388173a8443.png)
